.github/workflows/build-ffmpeg.yml

```diff
@@ -241,9 +244,13 @@ jobs:
       - name: patch
         working-directory: ffmpeg_sources/ffmpeg
         run: |
-          git apply -v --ignore-whitespace ../../ffmpeg_patches/ffmpeg/01-amf-colorspace-v2.patch
-          git apply -v --ignore-whitespace ../../ffmpeg_patches/ffmpeg/02-idr-on-amf.patch
-          git apply -v --ignore-whitespace ../../ffmpeg_patches/ffmpeg/03-amfenc-disable-buffering.patch
+          # AMD patches
+          git apply -v --ignore-whitespace ../../ffmpeg_patches/ffmpeg/*.patch
```

Comment:
I'm not sure if I like the wildcard usage. Although bash glob expansion should be alphabetical, it seems better to  patch individually so that we're guaranteed it will patch in the correct order, or if we need to selectively omit certain patches for targets in future, etc.